src/models/methods/accountCurrencies.ts

```diff
@@ -0,0 +1,21 @@
+import { BaseRequest, BaseResponse } from './baseMethod'
+import { LedgerIndex } from "../common"
+  
+export interface AccountCurrenciesRequest extends BaseRequest {
+    command: "account_currencies"
+    account: string
+    destination_account?: string
+    ledger_hash?: string
```

Comment:
can we put 

```ts
ledger_hash?: string
ledger_index?: index
```

In `baseTransaction.ts`? IIRC, its a parameter that is allowed in all public rippled methods.

Reply:
hm looks like you're right (though I'd want to dig into this a bit more). We might want to also open an issue on `xrpl-dev-portal` to add that to the "Base Request Format" page.

Reply:
Found a counter-example: https://xrpl.org/subscribe.html